### PR TITLE
Allow eventValue to be passed to GTM

### PR DESCRIPTION
**Pull request by brizido**
This ticket:

- This allows the eventLabel to be sent to GTM

### CODE REVIEW

File: lib/analytical/modules/google_tag_manager.rb [hunk 1]
```diff
@@ -28,19 +28,20 @@ def init_javascript(location)
...
+
           gtmVariables.event = "Integration GA-GTM Event";
           gtmVariables['eventCategory'] = options['eventCategory'];
           gtmVariables['eventAction'] = options['eventAction'];
           gtmVariables['eventValue'] = options['eventValue'];
-          
+
           try {
             var eventLabels = JSON.parse(options['eventLabel']);
+            gtmVariables['eventValue'] = options['eventValue'];
```
Comment (mariana-s-fernandes) [inline lib/analytical/modules/google_tag_manager.rb:39]: isn't this supposed to be "eventLabel" here @brizido ?

Comment (ciromiranda) [reply lib/analytical/modules/google_tag_manager.rb:39]:
```suggestion
            gtmVariables['eventLabel'] = options['eventLabel'];
```
Yes. I think it is what Mariana is saying.

**Comment by mariana-s-fernandes** [comment]
The revenue and brand team will deal with this

**Comment by jonasxiv** [comment]
@ciromiranda do you know how we can test this ?

**Review by mariana-s-fernandes — APPROVED** [review]
This makes sense. The event Label is always sent to GTM. And then, we try to parse json parameters from the event label in try/except clause. If it's a json, it should send other parameters as for example investment_id, campaign_id to GTM.

**Pull request merged**